prometheus_juju_backup_all_exporter/collector.py

```diff
@@ -47,7 +56,7 @@ def _fetch(self):
     ```
     """
 
-    def __init__(self, *metrics):
+    def __init__(self, *metrics: Any) -> None:
```

Comment:
can `metrics` really be `Any`? In the `_update_metrics` method, we expect items in `self._metrics` to have `_labelnames` attribute which Implies that we expect certain types of objects here.

Reply:
All of these issuse are caused by "hacking" the `Metric` class from the prometheus client, which they also did that using `_labelnames`. I think the complaints from `mypy` are reasonable and should be followed as a good practice. I tried to fixed that, but it's likely that I need to refactor / re-design to get that work. I will open another PR to address these issues from `mypy` later.